### DISCUSSION TITLE
Lodestar Standup 2024-13 (April 2, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#6615] [Standups]
### Meeting Info
[Apr 2, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/apr-2-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Planning Items
  - v1.18.0 milestones
  - Builder Boost Poll Results
- Team Updates
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by ensi321** [comment]
Would love to see an update of the team's opinion has changed regarding having 90 vs 100 as default boost factor #6568

**Reply by philknows** [reply]
Will hope to keep this short on the call this time now that we have [a bit more context](https://twitter.com/lodestar_eth/status/1772679499928191044). This will also leak into discussions about how we handle features like https://github.com/ChainSafe/lodestar/issues/6602, but it seems likely other clients will implement, but no consensus as to what this default value would be on each client AFAIK.

**Comment by philknows** [comment]
Would also like to get some feedback on our priorities in relation to client compatibility. There's been [efforts made by the EF DevOps ](https://github.com/kurtosis-tech/ethereum-package?tab=readme-ov-file#beacon-node--validator-client-compatibility)and proactively from other teams which have started uncovering some compatibility issues between our clients.

In addition, cross-compatibility with things like Rocketpool, DVTs and Vouch should be discussed as those types of issues net us users coming in through their software.

Some recent examples here:
https://github.com/ChainSafe/lodestar/pull/6576
https://github.com/ChainSafe/lodestar/pull/6593
https://github.com/ChainSafe/lodestar/issues/6564
https://github.com/ChainSafe/lodestar/issues/6401
https://github.com/ChainSafe/lodestar/issues/6419


Would like to understand our priority in resolving these and how we can do our part in ensuring as much compatibility if possible. Does it require some larger effort like #6518 to be implemented first?

**Comment by philknows** [comment]
And we're gonna do some quick open PR updates for monthly PR scrub. Noticeably high right now as our main scrubmaster is away for another 2 weeks.

**Comment by nflaig** [comment]
We should consider including https://github.com/ChainSafe/lodestar/pull/6556 in next release despite the memory leak which seems unrelated to peer count increase anyways.

Been running my node with 100 peers for a while now and based on my observations it slightly improves effectiveness. And I have seen users either increment this themselves or [are asking about it](https://discord.com/channels/593655374469660673/593655641445367808/1224711139368243290).

**Comment by twoeths** [comment]
maybe turn it on for some nodes and monitor for 1 release? not comfortable to see we have memory issues with it (even through it's not the root cause, we more likely see it with 100 target peers) and still release

**Comment by philknows** [comment]
Closed for #6653
